lib/mat8bit.gi

```diff
@@ -436,6 +436,69 @@ end);
 
 #############################################################################
 ##
+#M  CopyToMatrixRepNC( <list>, <fieldsize )
+#M  CopyToMatrixRep( <list>[, <fieldsize> | <field>])
+##
+
+
+InstallGlobalFunction(CopyToMatrixRep,
+        function( m,q )
+    local  newm;
+
+    
+    
+    
+    if not IsInt(q) then
+        if IsField(q) then
+            if Characteristic(q) = 0 then
+                return fail;
+            fi;
+            q := Size(q);
```

Comment:
What if `q` is a rational function field over a finite field? Then its characteristic is non-zero, yet its size is infinite. And we wouldn't know what to do with the matrix anyway. So perhaps use `IsFFECollection(q)` instead?